config/config.json.tmpl

```diff
@@ -1,9 +1,9 @@
 {
   "development": {
-    "username": "__YOUR_DEV_DB_USERNAME_HERE__",
-    "password": "__YOUR_DEV_DB_PASSWORD_HERE__",
+    "username": "user",
```

Comment:
Was there a reason to replace the prominent all-caps fill-these-in placeholders with the smaller "user" and "pass"?

Reply:
Oh, is that maybe a standard in Docker configuration -- something looks for those specific strings to replace?

Reply:
I'm using "user" and "pass" as the default user and pw for postgres in docker-compose.yml

Reply:
Gotcha.  And that's okay because this is all inside Docker and has no security implications for anyone else (right?  just checking).

However, `config/config.json.tmpl` is also a file people are supposed to edit (after copying it to `config/config.json`) when they're setting up a production service.  This change replaces the long, all-caps placeholders with short, lower-case strings, and furthermore the new strings aren't just placeholders -- they're actually used (within Docker).

So I *think* a better change would be to script the copying of `config/config.json.tmpl` to `config/config.json` and the replacement of the two placeholder parameters with the desired actual parameters, as part of Docker setup.  In other words, instead of changing the template config file, script the process of converting the template config file to the live file, so that the distinction between template and live file remains clear.

Does this make sense?

Reply:
Yeah, I see what you mean. I think it might be better to avoid editing JSON in a makefile and instead rely on a .env file for configuration? That way, docker-compose can read the parameters from there. I checked and config.json is only used in a couple of places so it shouldn't be hard to replace.

Reply:
Yep, all done. Config is consolidated in a .env file now